seed_helen_comments.py

```diff
@@ -0,0 +1,354 @@
+    # print "box", box
+
+    left_bg = ((0) * width)
+    top_bg = ((0) * height)
+    width_bg = ((.25) * width)
+    height_bg = ((.75) * height)
+    bg_box = (left_bg, top_bg, left_bg+width_bg, top_bg+height_bg)
+    # print "bg_box", bg_box
+
+    crop = img.crop(box)
+    bg_crop = img.crop(bg_box)
+    return get_colors(crop, bg_crop)
+
+
+def pillow_loop(img_urls):
+    # for url in img_urls:
+    # print "URL:", img_urls
+    response = requests.get(img_urls)
+    img = Image.open(StringIO(response.content))
+    # print "heading to crop"
+    return crop_image(img)
+
+
+def img_urls(show_url):
+    # this returns a list of images for a show
+    image_urls = set()
+
+    # print "getting: %s" % show_url
+    r = requests.get(show_url)
+    if r.status_code == 200:
+        print "got a hit!"
+        html_body = r.text.split(",")
+        for l in html_body:
+            match = re.match(r'.*(http:.*/_(?!ARC|AG|UMB).*.jpg).*', l)
+            # match = re.match(r'.*(http:\/\/assets.vogue.com\/photos\/.*\/master\/pass\/.*.[jpg|JPG]).*', l)
+            if match:
+                image_urls.add(match.group(1))
+    # print "image urls:", image_urls
+    # print len(image_urls)
+    # image_urls = list(image_urls)
+    return list(image_urls)
+    # pillow_loop(image_urls)
+
+
+def feed_urls():
+    # main fxn to call url generator and aggregate top colors by show
+    generated_urls = {year: {season: {brand: 'http://www.vogue.com/fashion-shows/{}-{}-ready-to-wear{}'.format(season, year, brand_url) for brand, brand_url in brands.items()} for season in seasons} for year in years}
+    # print "generated urls:", generated_urls
+    for year in generated_urls.keys():
+        for season in generated_urls[year].keys():
+            for brand, brand_url in generated_urls[year][season].items():
+                print "generated urls", generated_urls
+                print "brand url:", brand_url
+                print "season", season
+                print "brand name:", brand
+                show = load_show(year, season, brand)
+                image_urls = img_urls(brand_url)
+
+                top_show_colors = []
+                for url in image_urls:
+                    final_named_colors = pillow_loop(url)
+                    for color in final_named_colors:
+                        top_show_colors.append(color)
+                print "at feed urls, top show colors: ", top_show_colors
+
+                count_colors = {color: top_show_colors.count(color) for color in top_show_colors}
+                top_show_colors = sorted(set(top_show_colors), key=count_colors.get, reverse=True)
+                top_show_colors = top_show_colors[:10]
+                load_show_colors(top_show_colors, brand, year, season)
+                # jus have to pass top show color & show (or just show.show_id)
+
+
+def load_brands(brands):
+    """Load brands into database."""
+
+    print "Brands"
+
+    for key, value in brands.items():
```

Comment:
`for brand_name in brands.keys()`

Are you not going to save your route for each brand anywhere?